### ISSUE TITLE
how to train?

### ISSUE THREAD

**Issue by jianlong-yuan** [chenxi116/PNASNet.TF#2]
Could you give us your train script?

**Comment by chenxi116** [comment]
This repo is a simplified version of the official implementation in [TF Slim](https://github.com/tensorflow/models/tree/master/research/slim#Pretrained). If you want to train, I suggest using the official implementation. [Here](https://github.com/tensorflow/models/blob/master/research/slim/nets/nasnet/pnasnet.py#L34) may be a good place to start.

**Comment by jianlong-yuan** [comment]
Hi Thank you for your answer. But TF Slim also can not reproduce the results of your article by using RL. I don't find it uses RL.

**Comment by chenxi116** [comment]
I assume by training, you mean training our found architecture, PNASNet-5. I assume by RL, you mean reinforcement learning. First, training a network does not involve reinforcement learning: it uses stochastic gradient descent. Second, our search algorithm does not use reinforcement learning either.

**Comment by jianlong-yuan** [comment]
I apologize for my statement. I want to search for network structures, not structures you've already searched for.

**Comment by chenxi116** [comment]
I see. Sorry we did not release the code for architecture search, as that part is too heavily dependent on infrastructure.
